src/System.Security.Cryptography.Csp/src/System/Security/Cryptography/DESCryptoServiceProvider.cs

```diff
@@ -16,6 +16,7 @@ public sealed class DESCryptoServiceProvider : DES
 
         public DESCryptoServiceProvider() { }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA5351", Justification = "This is the implementation of DES")]
```

Comment:
It is possible. The SuppressMessageAttribute has a Target property that works i think. The GlobalSuppressions.cs that the IDE makes uses it.